src/contracts/helpers/Sorter.sol

```diff
@@ -103,8 +105,8 @@ contract Sorter {
 
         SortingData[] memory sortingData = new SortingData[](count);
 
-        bytes32 userOpHash =
-            dConfig.callConfig.allowsTrustedOpHash() ? userOp.getAltOperationHash() : userOp.getUserOperationHash();
+        address verification = IAtlas(userOp.to).VERIFICATION();
```

Comment:
We should probably make `IAtlasVerification VERIFICATION` an immutable set in the constructor (note, there was that issue recently about making these immutable address vars just the interface type e.g. IAtlas so we don't need to cast each time in the code)

Reply:
386b077